admin_scripts/ubuntu/ubuntu_16.04/change_superuser_password.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash 
+
+# This script will change the superuser password on a BibOS machine.
+
+
+# Expect exactly two input parameters
+
+if [ $# -ne 2 ]
+then
+    echo "usage: $(basename $0) <password> <confirmation>"
+    echo ""
+    exit -1
+fi
+
+if [ "$1" == "$2" ]
+then
+    # change password
+    TARGET_USER=superuser
+    PASSWORD="$1"
+    CRYPTPASS=$(perl -e 'print crypt($ARGV[0], "password")' $PASSWORD)
+    # Done calculating, now do it.
+    /usr/sbin/usermod $TARGET_USER -p $CRYPTPASS
```

Comment:
Er... isn't this a bit of a security issue? There's a tiny window in which users might be able to see this script running, and `crypt(3)` is a _terrible_ encryption function.